draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1406,10 +1406,13 @@ user agent MUST process the cookie as follows:
 3.  If the cookie-name or the cookie-value contains a %x00-1F / %x7F (CTL)
     character, abort these steps and ignore the cookie entirely.
 
-4.  Create a new cookie with name cookie-name, value cookie-value. Set the
+4.  If the sum of the lengths of cookie-name and cookie-value is more than
+    4096 bytes, abort these steps and ignore the cookie entirely.
```

Comment:
Recommend using the term "octet(s)" instead of "byte(s)" to match other usages. 
The line under 6.1 should also be updated.

Reply:
I don't have strong opinions on bytes vs octets, but if you change it, it's probably worth changing for the other length limits (in parsing).

Reply:
Thanks, "octets" is probably better since elsewhere it says
```
NOTE: Despite its name, the cookie-string is actually a sequence of
   octets, not a sequence of characters.
```

Reply:
Thanks, and you can ignore my comment about 6.1 I was looking at an outdated version.